vslib/src/sai_vs_hostintf.cpp

```diff
@@ -21,7 +21,11 @@
 #include <net/if_arp.h>
 #include <linux/if_ether.h>
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wshadow"
```

Comment:
it's separated from this PR;
meanwhile, i'm wondering if we want to upgrade the version of this `json.hpp` file to a newer one.

Reply:
if i remember there were some compiling issues with newer one thats why we stick with this one but you can take a look, im adding this since gcc 9.1 is complaining